app/Listeners/PublicAdministrationEventsSubscriber.php

```diff
@@ -76,7 +76,7 @@ public function onActivated(PublicAdministrationActivated $event): void
         $user->sendPublicAdministrationActivatedNotification($publicAdministration);
 
         logger()->notice(
-            'Public Administration ' . $publicAdministration->info . ' activated',
+            'Public Administration ' . $publicAdministration->name . ' activated',
```

Comment:
Come mai questo cambio di attributo? L'accesso a `info` è stato fixato in a4f3f3c286945578c0db111717ea639491b32b53.

Reply:
Faccio un rebase e sistemo